dbt_server/logging.py

```diff
@@ -91,92 +89,10 @@ def configure_uvicorn_access_log():
     ual.propagate = True
     ual.handlers = []
 
-
-json_formatter = dbt_logger.JsonFormatter(format_string=dbt_logger.STDOUT_LOG_FORMAT)
-
-
 @dataclass
 class ServerLog:
     state: TaskState
     error: Optional[str]
 
     def to_json(self):
         return json.dumps(self.__dict__)
-
```

Comment:
This logic was only implemented in unused endpoints, and has been changed for 0.2.0